tests/Main.elm

```diff
@@ -0,0 +1,13 @@
+port module Main exposing (..)
```

Comment:
Is there a specific reason why  we need `port` here? I thought we could run tests without `port` (if you want, take [this simple project as an example](https://github.com/cuducos/cunhajacaiu/blob/master/tests/frontend/Tests.elm)).